### PR TITLE
support any types for query as in

### PR DESCRIPTION
**Pull request by chiaen**
as in https://github.com/zabawaba99/firego/pull/65

### CODE REVIEW

File: query.go [hunk 1]
```diff
@@ -65,22 +66,23 @@ func (fb *Firebase) OrderBy(value string) *Firebase {
...
 }
 
-func escapeString(s string) string {
-	_, err := strconv.ParseInt(s, 10, 64)
-	if err != nil {
-		return `"` + strings.Trim(s, `"`) + `"`
+func escapeParameter(s interface{}) string {
+	switch s.(type) {
+	case string:
+		return fmt.Sprintf(`%q`, strings.Trim(s.(string), `"`))
```
Comment (FrankChung) [inline query.go]: "%q"

File: query.go [hunk 2]
```diff
@@ -65,22 +66,23 @@ func (fb *Firebase) OrderBy(value string) *Firebase {
...
+	default:
+		return fmt.Sprintf(`%v`, s)
```
Comment (FrankChung) [inline query.go]: "%v"

File: query.go (hunk 1 above)
Comment (FrankChung) [inline query.go]: "%q"

File: query_test.go [hunk 3]
```diff
@@ -159,9 +159,9 @@ func TestQueryMultipleParams(t *testing.T) {
 	)
```
Comment (FrankChung) [inline query_test.go:159]: can you pass the unit test? seems there are something to be changed.

Comment (chiaen) [reply query_test.go:159]: ok now

**Pull request merged**